pliers/extractors/audio.py

```diff
@@ -618,3 +634,67 @@ def _extract(self, stim):
         return ExtractorResult(preds, stim, self, features=labels,
                                onsets=onsets, durations=[dur]*len(onsets),
                                orders=list(range(len(onsets))))
+
+class MFCCLowEnergyExtractor(MFCCExtractor):
+    ''' Given a Middle Bound, Extract the Energy from the Low Register
+    of Mel-Frequency Ceptral Coefficients.
+
+    Args:
+    n_mfcc (int): specifies the number of MFCC to extract
+    n_coefs(int): cepstrum coefficients to keep in the low quefrency spectrum
+    hop_length (int): hop length in number of samples
+    librosa_kwargs (optional): Optional named arguments to pass to librosa
+    '''
+
+    _log_attributes = ('hop_length', 'n_coefs', 'n_mfcc')
+
+    def __init__(self, n_mfcc=48, n_coefs=13, hop_length=1024, **librosa_kwargs):
+        self.n_mfcc = n_mfcc
+        self.n_coefs = n_coefs
+        self.hop_length = hop_length
+        self.librosa_kwargs = librosa_kwargs
+        super().__init__(n_mfcc=n_mfcc)
+
+    @override
+    def _proc_func(self,stim):
+        vals = self._get_values(stim)
+        low_mfs = fft.dct(np.transpose(vals[:self.n_coefs]), 
```

Comment:
I think I prefer only having a single "EnergyExtractor". It seems too redundant to me to duplicate the logic of the low and high extractors. If we ever had to make any changes to this logic, we'd have to do it twice.

The only difference between the two extractors is this single line. 

How about instead, we add a parameter (`direction`?) that dedicates if we are taking `n_coefs` from the `high` or `low` end of the spectrum. If the user wants both, they simply run the same extractor twice with different parameters.